roles/contiv_network/tasks/main.yml

```diff
@@ -21,33 +36,33 @@
   copy: src=netmaster.service dest=/etc/systemd/system/netmaster.service
   when: run_as == "master"
 
-- name: start netmaster
-  shell: systemctl daemon-reload && systemctl start netmaster
+- name: restart netmaster
+  shell: systemctl daemon-reload && systemctl stop netmaster && systemctl start netmaster
   when: run_as == "master"
 
 - name: copy environment file for netplugin
-  copy: src=netplugin dest=/etc/default/netplugin
+  template: src=netplugin.j2 dest=/etc/default/netplugin mode=0644
 
 - name: copy systemd units for netplugin
   copy: src=netplugin.service dest=/etc/systemd/system/netplugin.service
 
-- name: start netplugin
-  shell: systemctl daemon-reload && systemctl start netplugin
+- name: restart netplugin
+  shell: systemctl daemon-reload && systemctl stop netplugin && systemctl start netplugin
 
 - name: setup netmaster host alias
   shell: echo "{{ service_vip }} netmaster" >> /etc/hosts
 
 # XXX: need to move the following to correct roles
-- name: install contivctl
-  shell: source /etc/profile.d/00golang.sh && go get github.com/contiv/contivctl
-
-- name: install docker-compose
-  shell: pip install docker-compose
-
-- name: install contiv-compose
-  shell: >
-     source /etc/profile.d/00golang.sh && \
-     go get github.com/contiv/libcompose && \
-     cd $GOPATH/src/github.com/contiv/libcompose && \
-     make binary && \
-     ln -s `pwd`/bundles/libcompose-cli_linux-amd64 $GOPATH/bin/contiv-compose
+#- name: install contivctl
```

Comment:
+1 to remove this stuff if it is not needed. This was added as a quick solution to get dockercon demo setup going.

In future, if we need `contiv-compose` again we shall first introduce released versions for it and use them instead.